src/rules/no-named-default.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    function checkSpecifiers(type, node) {
+      if (node.source == null) return
+
+      const hasImportSpecifier = node.specifiers.some(function (im) {
+        return im.type === type
+      })
+
+      if (!hasImportSpecifier) {
+        return
+      }
+
+      node.specifiers.forEach(function (im) {
+        if (im.type !== type) return
+
+        if (im.imported.name === 'default') {
+          context.report(im.local,
+            'Using name \'' + im.local.name +
+            '\' as identifier for default export.')
```

Comment:
```js
context.report({
  node: im.local,
  message: 'Using name...'
})
```

Both syntaxes are valid but I think the first one was deprecated. Anyhow, this would be more consistent with the other rules in this repo AFAIK.